.golangci.yaml

```diff
@@ -0,0 +1,81 @@
+run:
+  tests: true
+
+linters-settings:
+  funlen:
+    lines: 150
+    statements: 80
+  depguard:
+    rules:
+      main:
+        # List of file globs that will match this list of settings to compare against.
+        # Default: $all
+        files:
+          - "!**/*_a _file.go"
+        # List of allowed packages.
+        allow:
+          - $gostd
+          - github.com/yakuninmax/imgpreviewer
+          - github.com/pkg/errors
+          - github.com/stretchr/testify/require
+          - github.com/disintegration/imaging
+  dupl:
+    threshold: 160
+
+linters:
+  disable-all: true
+  enable:
+    - asciicheck
+    - bodyclose
+    #- deadcode
```

Comment:
С закоменченными линтерами такая проблема:

WARN The linter 'varcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused. 
WARN The linter 'deadcode' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused. 
WARN The linter 'exportloopref' is deprecated (since v1.60.2) due to: Since Go1.22 (loopvar) this linter is no longer relevant. Replaced by copyloopvar. 
WARN The linter 'ifshort' is deprecated (since v1.48.0) due to: The repository of the linter has been deprecated by the owner.  
WARN The linter 'structcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused. 
ERRO [linters_context] deadcode: This linter is fully inactivated: it will not produce any reports. 
ERRO [linters_context] ifshort: This linter is fully inactivated: it will not produce any reports. 
ERRO [linters_context] structcheck: This linter is fully inactivated: it will not produce any reports. 
ERRO [linters_context] varcheck: This linter is fully inactivated: it will not produce any reports.

Поэтому пришлось их отключить.